### ISSUE TITLE
Missing `current` directory

### ISSUE THREAD

**Issue by tacerus** [FiskFan1999/wvlist#5]
Hi!

Thanks for the project!

I built it using `go 1.17.6` from the current Git master branch, running it using `./wvlist run -c config.json` works, however upon trying to load the website it resets the connection and prints:

```
open current: no such file or directory                                                                                                           
2022/02/25 03:50:42 http: panic serving 192.168.5.10:59860: runtime error: slice bounds out of range [:9] with length 0
goroutine 18 [running]:                                                  
net/http.(*conn).serve.func1()                                                                                                                    
        /usr/lib64/go/1.17/src/net/http/server.go:1802 +0xb9                                                                                      
panic({0x74fa60, 0xc000206030})                                          
        /usr/lib64/go/1.17/src/runtime/panic.go:1047 +0x266                                                                                       
main.GetLinkToCommitInRepositry({0x0, 0x55})                                                                                                      
        /mnt/WD01/GitHub/wvlist/linkToRepo.go:28 +0x155
main.HomePage({0x800cd8, 0xc000228000}, 0xc00021c000)
        /mnt/WD01/GitHub/wvlist/homePage.go:51 +0x207
net/http.HandlerFunc.ServeHTTP(0x0, {0x800cd8, 0xc000228000}, 0x0)
        /usr/lib64/go/1.17/src/net/http/server.go:2047 +0x2f
net/http.(*ServeMux).ServeHTTP(0x0, {0x800cd8, 0xc000228000}, 0xc00021c000)
        /usr/lib64/go/1.17/src/net/http/server.go:2425 +0x149
net/http.serverHandler.ServeHTTP({0xc000210090}, {0x800cd8, 0xc000228000}, 0xc00021c000)
        /usr/lib64/go/1.17/src/net/http/server.go:2879 +0x43b
net/http.(*conn).serve(0xc000214000, {0x802140, 0xc00007d3e0})
        /usr/lib64/go/1.17/src/net/http/server.go:1930 +0xb08
created by net/http.(*Server).Serve
        /usr/lib64/go/1.17/src/net/http/server.go:3034 +0x4e8
```

Did I miss something, or is it not compatible yet? I think the project is quite new - apologies I am too early!

Best,
Georg

**Comment by tacerus** [comment]
Apologies! I accidentally missed the Makefile and wrongly initialized the modules myself.

**Comment by FiskFan1999** [comment]
Hey @tacerus, thanks for being here! I just wanted to let you know that you will also need to create the following new directories: (unfortunately these will lead to panics while running)

- `current` which you already have created
- `lilypond`
- `submissions`
I'm going to try fix this bug straightaway, by creating the directories automatically if not present.

**Comment by FiskFan1999** [comment]
tacerus, I just pushed e811bd02e288c2502a29b538daec3dd5419bf21a which _should_, I hope, create those directories which are required when the binary is run on a fresh install. Will tag this v1.0.2 tomorrow.

**Comment by tacerus** [comment]
Hi @FiskFan1999! Thank you very much! I stumbled across the `submissions` one upon trying to submit, but didn't notice the need for `lilypond` yet. :-) I'll update and try having it create the directories for me.

**Comment by FiskFan1999** [comment]
Hey @tacerus, I just released this and another bugfix in v1.0.2, thank you for the report.
